docs/Setup.md

```diff
@@ -820,11 +820,11 @@ Default value: 'TB'
 
 | Parameter     | Description                                                 | Type    | Required | Values             |
 | ------------- | ----------------------------------------------------------- | ------- | -------- | ------------------ |
-| entityPadding | Minimum internal padding betweentext in box and box borders | Integer | 4        | Any Positive Value |
+| entityPadding | Minimum internal padding between text in box and box borders | Integer | 4        | Any Positive Value |
```

Comment:
How did you fix this content ? It’s generated from src/mermaidAPI.js src/config.js src/defaultConfig.js so please fix the js files first and then generate it with postbuild task : https://github.com/mermaid-js/mermaid/blob/ef9740bb3816fe211122475ba5a617a506dadc9f/package.json#L27